shared/ssa/qlpack.yml

```diff
@@ -1,4 +1,4 @@
 name: codeql/ssa
-version: 0.0.2-dev
+version: 0.0.2
```

Comment:
You're bumping the version of the shared packs.   
But you're not bumping the version in the `.qlpack`s that depend on these shared packs?